.travis.yml

```diff
@@ -1,15 +1,50 @@
 language: go
 go:
- - "1.10.x"
+  - "1.10.x"
+
+addons:
+  ssh_known_hosts: ec2-13-59-124-240.us-east-2.compute.amazonaws.com
+
+before_install:
+  - export VITRINESOCIAL_ENV=production
+  - mkdir server/tmp # create storage directory
+  - openssl aes-256-cbc -K $encrypted_c41c576030a0_key -iv $encrypted_c41c576030a0_iv -in vitrinesocial.pem.enc -out devops/vitrinesocial.pem -d #decrypt aws key
+  - chmod 400 devops/vitrinesocial.pem
+
+install:
+  - make install
+  - cd frontend
+  - yarn install
+  - cd .. #back to root directory
+
+cache:
+  directories:
+    - "node_modules"
 
 script:
- - export VITRINESOCIAL_ENV=test
- - mkdir server/tmp # create storage directory
- - make install
- - make coverage
+  - make coverage
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then make build; fi'
+  - make build
+  - cd frontend
+  - yarn test
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then yarn build; fi'
```

Comment:
esse if não esta fazendo nada neh?

Reply:
É para estar fazendo o build apenas quando for pull request.

Reply:
é que ele chama em baixo igual não?

Reply:
ah sim, eu tinha deixado dois, um para o build do back e outro para o build do front. Já rearranjei no mesmo if.